packages/hoppscotch-app/locales/tw.json

```
@@ -245,11 +245,11 @@
     "communication": "通訊",
     "log": "日誌",
     "message": "訊息",
-    "publish": "釋出",
+    "publish": "發佈",
```

```suggestion
    "publish": "發布",
```
Ref: https://terms.naer.edu.tw/detail/16267516/?index=7
[Android](https://cs.android.com/android/platform/superproject/+/master:packages/services/Car/service/res/values-zh-rTW/strings.xml;l=87)
[GNOME](https://gitlab.gnome.org/GNOME/evolution/-/blob/master/po/zh_TW.po#L22270)

---

But [Microsoft](https://www.microsoft.com/en-us/language/Search?&searchTerm=publish&langID=129&Source=true&productid=0), [Apple](https://support.apple.com/zh-tw/HT208716), and [Adobe](https://helpx.adobe.com/tw/indesign/using/publish-online.html) uses 發佈。